From b7b69cdcdb4e2d80a85e303cc3230666d565bc39 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Sun, 11 Oct 2015 09:14:52 +0200
Subject: [PATCH] Fix typo for arm predefined macro in
 atomicops_internals_arm_gcc.h

Fix typo  __ARM_ARCH_6KZ__ vs.  __ARM_ARCH_6ZK__, fixes
compile for raspberry pi.

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 deps/v8/src/base/atomicops_internals_arm_gcc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/v8/src/base/atomicops_internals_arm_gcc.h b/deps/v8/src/base/atomicops_internals_arm_gcc.h
index 069b1ff..b924981 100644
--- a/deps/v8/src/base/atomicops_internals_arm_gcc.h
+++ b/deps/v8/src/base/atomicops_internals_arm_gcc.h
@@ -63,7 +63,7 @@ inline void MemoryBarrier() {
     defined(__ARM_ARCH_7R__) || defined(__ARM_ARCH_7M__) || \
     defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_6J__) || \
     defined(__ARM_ARCH_6K__) || defined(__ARM_ARCH_6Z__) || \
-    defined(__ARM_ARCH_6KZ__) || defined(__ARM_ARCH_6T2__)
+    defined(__ARM_ARCH_6ZK__) || defined(__ARM_ARCH_6T2__)
 
 inline Atomic32 NoBarrier_CompareAndSwap(volatile Atomic32* ptr,
                                          Atomic32 old_value,
-- 
2.1.4

